control/control_test.go

```diff
@@ -2,20 +2,30 @@ package control
 
 import (
 	"fmt"
-	"io/ioutil"
 	"math/rand"
 	"net"
-	"net/http"
 	"os"
 	"path/filepath"
-	"strings"
 	"testing"
 	"time"
 
+	"github.com/joyent/containerpilot/events"
 	"github.com/joyent/containerpilot/tests"
 	"github.com/joyent/containerpilot/tests/assert"
 )
 
+type TestApp struct {
```

Comment:
Test plumbing can be removed if its not used. I believe I was going to mock out the App interface and test that the endpoint handlers at least fired.